rwengine/src/objects/VehicleObject.hpp

```diff
@@ -6,7 +6,7 @@
 #include <objects/VehicleInfo.hpp>
 
 class CollisionInstance;
-class btVehicleRaycaster;
+struct btVehicleRaycaster;
```

Comment:
These are declared in `btBulletDynamicsCommon.h` or `btVehicleRaycaster.h` etc.
I think I remember a discussion with other projects about this forward declaration / protoyping being better though?

Can someone clarify this for me or shall we include here?